nano/core_test/node.cpp

```diff
@@ -1850,7 +1850,11 @@ TEST (node, rep_remove)
 	}
 
 	// Create inactive TCP channel for Rep1
-	std::shared_ptr<nano::transport::channel> channel_rep1 (std::make_shared<nano::transport::channel_tcp> (searching_node, std::weak_ptr<nano::socket> ()));
+	nano::tcp_endpoint endpoint_rep1 (boost::asio::ip::address_v6::loopback (), nano::test_node_port ());
```

Comment:
Can we just use an inproc channel? Like here:
https://github.com/nanocurrency/nano-node/commit/91ef65fc2a37c8bad69fff2c7b5356aa4a23f886

An outer node might also make sense here too. Not sure, I haven't really thought it through. about the outer node

Reply:
I'll try with an inproc channel. This should simplify things.